site/write_ups/2023/29-03-23_writeup.md

```diff
@@ -0,0 +1,127 @@
+
+
+We wondered if anything is really ethically neutral, especially in the domain of research. It is conceivable that something which seems unimportant now could have unintended side effects when it is put into different contexts. You can carry out your research, focussing on things you perceive as more “ethical” – whatever that means to you – but from an academic perspective, when you put something out there, it’s out.  Technologies we develop could have unforeseen long term effects, such as Huw’s Ph.D., which now has unclear implications, but what if 50 years from now someone uses it to make a bioweapon? Will Huw finally do something useful? 
+
+Network science is another example domain which might seem initially unsuspicious, yet there is a lot of defence funding that goes into the area, with military applications such as looking at social networks to detect terror cells. Similarly, in the education sector the integration of technology could result in biased and harmful prophecies for children. Teachers can have preconceptions of students from certain backgrounds, which may lead to models limiting those students’ access to resources by predicting what they might (not) achieve.
+
+Unfortunately, in domains like maths and physics, it is common for abstract models to be developed and then later down the line applied in harmful ways. If someone wants to build it, a lot of the time it is quite easy to do. At the very least, we thought you can avoid being the final step (i.e. the harmful application).
+
+
+*When do we decide when not to build something?*
+
+
+Considering the risk of unforeseen harms, at what point do you decide not to build technology? How far away from harm do you have to be down the causal chain to have responsibility for the consequences of your work? Is it when there are very clear implications that Huw’s work could be used for genocide, or something else awful, like a boring PhD thesis? How much harm will he cause his reviewers? Should he be stopped now?
+
+It is important to distinguish the point at which you halt development, from where mitigation techniques should be brought in. Maybe someone could develop some sort of signposting to help this... Something like [Data Hazards](https://datahazards.com/) labels? Just a thought.
+
+To understand where these thresholds are, the podcast talks about projects in the domain of high risk, high consequence. If there is a high probability that things could go wrong, and if things go wrong they go really wrong, maybe you should stop. [An article we read in a previous reading group](https://journals.sagepub.com/doi/10.1177/20539517221146122) provided a quite simple but useful heuristic for machine learning.  The paper suggests that ML should be used for well-defined, ‘learnable’, and low stakes problems, with relatively small ranges of subjectivity. This might help when we are trying to think about what could be done, or go wrong with, the models we build.
```

Comment:
Did we read that article in a previous Data Ethics Club meeting? If yes it would be cool to link the writeup too! But I'm not sure which one you meant